CorsixTH/Lua/languages/brazilian_portuguese.lua

```diff
@@ -57,23 +57,23 @@ confirmation.restart_level = "Você tem certeza de que deseja reiniciar o nível
 -- this is only relevant to the English game so does not need adding
 -- in other language files
 vip_names = {
- [6] = "O Presidente da Cruz Vermelha",
+ [6] = "Sir Lancelot Spratt",
```

Comment:
This can be a local celebrity figure or made up Brazilian name if it would make more sense. Up to you, just avoid controversy.